Source/Core/Common/TraversalProto.h

```diff
@@ -0,0 +1,96 @@
+// This file is public domain, in case it's useful to anyone. -comex
+
+#pragma once
+#include <array>
+#include "Common/CommonTypes.h"
+
+#define NETPLAY_CODE_SIZE 8
+typedef std::array<char, NETPLAY_CODE_SIZE> TraversalHostId;
```

Comment:
Because it's part of the packets, and the packets are fixed-length because the protocol was designed to be very simple to assist scalability.